tests/addressbook/external_test.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (C) 2017  Lucas Hoffmann
+# This file is released under the GNU GPL, version 3 or a later revision.
+# For further details see the COPYING file
+from __future__ import absolute_import
+
+import unittest
+
+import mock
+
+from alot.addressbook import external
+
+
+class TestExternalAddressbookGetContacts(unittest.TestCase):
+
+    """Some test cases for
+    alot.addressbook.external.ExternalAddressbook.get_contacts"""
+
+    regex = '(?P<name>.*)\t(?P<email>.*)'
```

Comment:
No, but it's being explict that this string is meant to be passed into one of the `re` functions. It's not the end of the world either way.